### ISSUE TITLE
Your platform is not compatible yet: darwin_x64

### ISSUE THREAD

**Issue by theasmoth** [NectarJS/nectarjs#58]
```
[!] Your platform is not compatible yet: darwin_x64
[!] Please open an issue: https://github.com/nectarjs/nectarjs
```

**Comment by hotmaycold** [comment]
```
[!] Your platform is not compatible yet: linux_x64
[!] Please open an issue: https://github.com/nectarjs/nectarjs
```

**Comment by adrien-thierry** [comment]
> ```
> [!] Your platform is not compatible yet: linux_x64
> [!] Please open an issue: https://github.com/nectarjs/nectarjs
> ```

@hotmaycold please open a new issue

**Comment by hotmaycold** [comment]
> > ```
> > [!] Your platform is not compatible yet: linux_x64
> > [!] Please open an issue: https://github.com/nectarjs/nectarjs
> > ```
> 
> @hotmaycold please open a new issue

done [issues/58](https://github.com/NectarJS/nectarjs/issues/58)

**Comment by hotmaycold** [comment]
> > > ```
> > > [!] Your platform is not compatible yet: linux_x64
> > > [!] Please open an issue: https://github.com/nectarjs/nectarjs
> > > ```
> > 
> > 
> > @hotmaycold please open a new issue
> 
> done [issues/59](https://github.com/NectarJS/nectarjs/issues/59)

**Comment by adrien-thierry** [comment]
Corrected v0.2.7
